installer/tue-install-impl.bash

```diff
@@ -280,8 +281,10 @@ function _try_branch_git
         tue-install-error "Invalid _try_branch_git: needs two arguments (repo and branch)."
     fi
 
-    tue-install-debug "git -C $1 checkout $2"
-    _try_branch_res=$(git -C "$1" checkout "$2" 2>&1) # This is a "global" variable from tue-install-git
+    repo="$1"
+    branch="$2"
```

Comment:
Use local variables?

Reply:
Apply this to `_try_branch_hg` too

Reply:
hg is not used in any target. Any reason not to remove the duplicate maintenance?

Reply:
I am not sure about dropping HG support yet.

Reply:
You know hg itself is deprecated?

Reply:
I know bitbucket dropped support. But HG itself is still develop, correct?

Reply:
Ah, I thought Atlassian 'owned' it.

Reply:
But still, is it worth it? Do you really expect a ROS hg source in the future?

Reply:
I explicitly removed hg, svn etc from Avular's fork as there is no point in maintaining something that's not being used anywhere

Reply:
I will remove all HG features.